Dockerfile

```diff
@@ -29,14 +29,15 @@ LABEL "nemo.library"=${IMAGE_LABEL}
 WORKDIR /opt
 
 # Install the minimal libcu* libraries needed by NeMo Curator
-RUN conda create -y --name curator -c conda-forge -c nvidia \
+RUN conda create -y --name curator -c nvidia/label/cuda-${CUDA_VER} -c conda-forge
   python=3.10 \
```

Comment:
Looks like gpuCI is currently throwing an error here.

Reply:
Apparently build arg's can't be in `RUN`, so had to create env var, which is ugly

Reply:
Oh lol, interesting. LGTM!